apps/framework-docs/src/pages/moose/moose-cli.mdx

```diff
@@ -154,6 +154,19 @@ moose generate migration --url https://<remote-env> --token <api-token> --save
 - Omit `--save` to output to stdout without writing files
 - Requires `[features].olap = true`; execution of saved plan requires `[features].ddl_plan = true`
 
+### DB Pull (External Tables)
+Refresh `EXTERNALLY_MANAGED` table definitions from a remote ClickHouse instance.
+```bash
+moose db pull --connection-string <CONNECTION_STRING> [--file-path <OUTPUT_PATH>]
+```
+- `--connection-string`: ClickHouse URL; native `clickhouse://` is auto-converted to HTTP(S). Include `?database=` or the CLI will query the current database.
+- `--file-path`: Optional override for the generated external models file (defaults to `app/externalModels.ts` or `app/external_models.py`).
+
+Notes:
+- Only tables marked `EXTERNALLY_MANAGED` in your code are refreshed.
+- The command writes a single external models file and creates a small git commit if changes are detected.
```

Comment:
is this normal behavior to automatically generate a git commit on behalf of the user? i feel like i'd want to create a commit myself only when i'm ready?